runtimes/php-8.0/composer.json.fallback

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "user/function",
+    "description": "",
+    "type": "library",
+    "license": "MIT",
```

Comment:
What is this file used for?

Reply:
Our PHP server MUST see `user/function` library. So if user does not have `composer.json`, we use this fallback one.

Reply:
If the user does not add their own composer file we add one for them so when we install dependencies and include the user's code as a package it doesn't crash

Reply:
oops, my GitHub didn't refresh with Matej's comment. My bad